### PR TITLE
Solution for fizzbuzz

### PR DESCRIPTION
**Pull request by hienhpss**

### CODE REVIEW

File: week0001/namhi.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+def fizzbuzz(n=100):
```
Comment (vietlq) [inline week0001/namhi.py]: This is a duplicate. Please remove  this file.

File: week0001/fizzbuzz_nam.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+def fizzbuzz(n=100):
+	loop = (x%15 for x in range(1,n+1) if x%3==0 or x%5==0)
+	for x in loop:
+		if x in (3,6,9,12):
+			yield "fizz"
```
Comment (vietlq) [inline week0001/fizzbuzz_nam.py]: `Fizz` and `Buzz`, please capitalize the first character

File: week0001/fizzbuzz_nam.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+def fizzbuzz(n=100):
+	loop = (x%15 for x in range(1,n+1) if x%3==0 or x%5==0)
```
Comment (vietlq) [inline week0001/fizzbuzz_nam.py]: Space around operators: `%`, `==`, `,`

File: week0001/namhi.py (hunk 1 above)
Comment (hienhpss) [inline week0001/namhi.py]: Fixed

File: week0001/fizzbuzz_nam.py (hunk 2 above)
Comment (hienhpss) [inline week0001/fizzbuzz_nam.py]: Done

File: week0001/fizzbuzz_nam.py (hunk 3 above)
Comment (hienhpss) [inline week0001/fizzbuzz_nam.py]: Done

**Comment by vietlq** [comment]
This solution is fine but a little complicated. You can follow up and add a simpler solution is you want to. I'm accepting this.

**Pull request merged**